tests/hyperv_test.go

```diff
@@ -146,7 +146,7 @@ var _ = Describe("[Serial][sig-compute] Hyper-V enlightenments", Serial, decorat
 				}
 			})
 
-			It("should be able to start successfully", func() {
+			It("should be able to start successfully", decorators.Reenlightenment, func() {
```

Comment:
I would place this decorator at the `Describe()` level since all of the tests are related to it

Reply:
Done.